### PR TITLE
tools/pydfu.py: Refactor syntax, update comments and docstrings

### PR DESCRIPTION
**Pull request by c0rejump**
I'm learning :)

### CODE REVIEW

**Comment by dpgeorge** [comment]
Thanks for the contribution.  It looks OK to me, the only thing I'd say is to relax the hard line-length limit.

File: tools/pydfu.py [hunk 1]
```diff
@@ -119,16 +127,17 @@ def init():
         status = get_status()
         if status == __DFU_STATE_DFU_IDLE:
             break
-        elif (status == __DFU_STATE_DFU_DOWNLOAD_IDLE
-            or status == __DFU_STATE_DFU_UPLOAD_IDLE):
+        elif (status == __DFU_STATE_DFU_DOWNLOAD_IDLE or
+              status == __DFU_STATE_DFU_UPLOAD_IDLE):
```
Comment (dpgeorge) [inline tools/pydfu.py]: I think it's clearer with the "or" on the second line.

File: tools/pydfu.py [hunk 2]
```diff
@@ -119,16 +127,17 @@ def init():
...
             abort_request()
         else:
             clr_status()
 
 
 def abort_request():
     """Sends an abort request."""
-    __dev.ctrl_transfer(0x21, __DFU_ABORT, 0, __DFU_INTERFACE, None, __TIMEOUT)
+    __dev.ctrl_transfer(0x21, __DFU_ABORT, 0, __DFU_INTERFACE,
+                        None, __TIMEOUT)
```
Comment (dpgeorge) [inline tools/pydfu.py]: This doesn't need to be split, we usually go for a maximum line length of "about 100".

**Comment by c0rejump** [comment]
Hi,

Thanks for all comments. PR updated.

**Comment by dpgeorge** [comment]
Thanks!  Rebased, squashed and merged in 6db5cede06dfb580c5e4f6cdeb1796256969b4fb

**Pull request closed without merging**